di_website/spotlight/models.py

```diff
@@ -3,11 +3,15 @@
 from wagtail.search import index
 from wagtail.core.models import Page
 from wagtail.core.fields import StreamField
-from wagtail.admin.edit_handlers import FieldPanel, MultiFieldPanel
+from wagtail.admin.edit_handlers import FieldPanel, MultiFieldPanel, StreamFieldPanel
 from wagtail.snippets.edit_handlers import SnippetChooserPanel
+from wagtail.core.blocks import (
+    StreamBlock
+)
```

Comment:
One line for this, no braces. It's a single import.

Reply:
Updated to one line